### PR TITLE
Academyv2 changes

### PR DESCRIPTION
**Pull request by CryptoManion**
Article update

### CODE REVIEW

**Review by yurikoinaba — APPROVED** [review]

**Review by Haxen2000 — APPROVED** [review]

**Comment by CryptoManion** [comment]
New updates on the academy. I need to know if I did well for the image (and everything else :))

**Review by yurikoinaba — APPROVED** [review]

**Comment by CryptoManion** [comment]
typos and links

File: docs/consensus/pos-vs-pow.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Proof of Work is one of the most secure consensus mechanisms but it is only secu
 
 **Game theory** is the study of mathematical models of strategic interactions between decision-makers. 
 
-Miners are rational decision-makers in a PoW blockchain. The incentives to act according to the rules of the [mining protocol] are embedded in the protocol itself, and outweigh the potential gains of behaving maliciously. 
+Miners are rational decision-makers in a PoW blockchain. The incentives to act according to the rules of the [mining protocol](mining/crypto-mining) are embedded in the protocol itself, and outweigh the potential gains of behaving maliciously.
```
Comment (yurikoinaba) [inline docs/consensus/pos-vs-pow.md:105]:
```suggestion
Miners are rational decision-makers in a PoW blockchain. The incentives to act according to the rules of the [mining protocol](crypto-mining.md) are embedded in the protocol itself, and outweigh the potential gains of behaving maliciously. 
```

File: docs/consensus/pos-vs-pow.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ Proof of Work is one of the most secure consensus mechanisms but it is only secu
 
 **Game theory** is the study of mathematical models of strategic interactions between decision-makers. 
 
-Miners are rational decision-makers in a PoW blockchain. The incentives to act according to the rules of the [mining protocol] are embedded in the protocol itself, and outweigh the potential gains of behaving maliciously. 
+Miners are rational decision-makers in a PoW blockchain. The incentives to act according to the rules of the [mining protocol](crypto-mining.md) are embedded in the protocol itself, and outweigh the potential gains of behaving maliciously.
```
Comment (yurikoinaba) [inline docs/consensus/pos-vs-pow.md:105]:
```suggestion
Miners are rational decision-makers in a PoW blockchain. The incentives to act according to the rules of the [mining protocol](mining/crypto-mining.md) are embedded in the protocol itself, and outweigh the potential gains of behaving maliciously. 
```

Comment (yurikoinaba) [reply docs/consensus/pos-vs-pow.md:105]: Apologies, I had made the incorrect suggestion previously

**Review by yurikoinaba — APPROVED** [review]

**Pull request merged**